frontend/src/static/js/components/test/webstatus-stats-missing-one-impl-chart-panel.test.ts

```diff
@@ -246,10 +246,12 @@ describe('WebstatusStatsMissingOneImplChartPanel', () => {
     expect(rows.length).to.equal(10, 'should have 10 rows');
 
     const firstRowCells = rows[0].querySelectorAll('td');
-    expect(firstRowCells[0].textContent?.trim()).to.equal(
-      'css',
-      'first row ID',
-    );
+    const textContent = firstRowCells[0].textContent
+      ?.split('\n')
+      .map(word => word.trim())
+      .filter(word => word.length > 0)
+      .join(' ');
```

Comment:
I don't know if this is the best way to go about confirming the text here, but there's a lot of white space and new-lines otherwise.

Reply:
We can always revisit in the future if needed.
LGTM